src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.html

```diff
@@ -1 +1,36 @@
-<div class="card"></div>
+<div class="list-habit-content">
+  <div class="list-container">
```

Comment:
Could be merged those 2 `div` elements? I really don't understand what is the reason to use
```html
<div>
  <div>
    ...
  </div>
</div>
```
If you had something like"
```html
<div>
  <div>
    ...
  </div>
  <div>
    ...
  </div>
</div>
```
it would be understandably